Source/WebCore/css/StyleSheet.idl

```diff
@@ -26,11 +28,12 @@
     JSCustomHeader,
     JSCustomMarkFunction,
 ] interface StyleSheet {
-    readonly attribute DOMString? type;
-    attribute boolean disabled;
+    readonly attribute DOMString type;
+    readonly attribute USVString? href;
+    // FIXME: Update 'ownerNode' to have (Element or ProcessingInstruction)? rather than 'Node'.
     readonly attribute Node ownerNode;
```

Comment:
I am able to make it work / compile with `(Node or ProcessingInstruction)`?. Should I do this change? @nt1m , @rniwa

Reply:
I have done this change and letting it run through EWS. Fingers crossed!

Reply:
No, `ProcessingInstruction` is a `Node`, so this just redundant. I would just leave the `FIXME`

Reply:
Should I add `?` because it can be nullable?

Reply:
Reverted my change but added '?' because web-spec says that it can be nullable. Slightly updated my FIXME as well to refer to web-specification.

Reply:
My guess is that it's nullable in the spec when the type matches neither Element or ProcessingInstruction ?

Reply:
Firefox / Gecko has nullable at 'Node': https://searchfox.org/mozilla-central/source/dom/webidl/StyleSheet.webidl#19

Reply:
Similarly in Chromium / Blink as well: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/css/style_sheet.idl;l=29;drc=0aee4434a4dba42a42abaea9bfbc0cd196a63bc1

Reply:
OK might be fine to change then.

Reply:
@nt1m - Thanks for your help and review. Really appreciate your insight.